mw-webapp/src/component/input/Input.tsx

```diff
@@ -3,6 +3,11 @@ import clsx from "clsx";
 import {InputMode} from "src/component/input/InputMode";
 import styles from "src/component/input/Input.module.scss";
 
+/**
+ * Max amount of characters in input with type "number"
+ */
+const MAX_AMOUNT_CHARACTERS = 4;
```

Comment:
why do we need to count characters for input number?

Reply:
Del